### PR TITLE
added trend line support for area chart

### PR DESCRIPTION
**Pull request by RihanaE**

### CODE REVIEW

File: src/Components/AreaChartComponent.js [hunk 1]
```diff
@@ -52,46 +51,92 @@ const AreaChartComponent = ({ chartData, chartInfo, item }) => {
     return { xData, seriesData };
   };
 
+  // Calculate trend line data based on regression type
+  const calculateTrendLine = (seriesData, regressionType) => {
+    const dataPoints = seriesData.map((value, index) => [index, value]);
+
+    let result;
+    switch (regressionType) {
```
Comment (redet-G) [inline src/Components/AreaChartComponent.js:59]: DHIS2 only supports linear, polynomial and loess regression. I don't see the loess regression here. You can copy it from the line chart it should be similar.

**Comment by RihanaE** [comment]
Okay, I will

**Comment by redet-G** [comment]
This is great but we are only drawing Area chart for the first serious only.
This is our graph: 
image
And, this is what it should look like
image

**Comment by redet-G** [comment]
Additionally, Can we make the trend line dotted similar to the line chart?

**Comment by redet-G** [comment]
LGTM!

**Pull request merged**